src/main/java/Lubot.java

```diff
@@ -0,0 +1,202 @@
+        System.out.println("lubot: type 'tasks' to see tasks");
+        System.out.println("lubot: type 'exit' to exit");
+        System.out.println("lubot: type 'mark <int>' to mark a task");
+        System.out.println("lubot: type 'delete <int>' to delete a task");
+        System.out.println("lubot: type 'todo' to add a todo");
+        System.out.println("lubot: type 'deadline /by <date>' to add a deadline");
+        System.out.println("lubot: type 'event /from <date> /to <date>' to add an event");
+        System.out.println(horizontalBar);
+
+        // main
+        Scanner scanner = new Scanner(System.in);
+        
+        while (true) {
+            System.out.print("You: ");
+            String userInput = scanner.nextLine();
+
+            // exit
+            if (userInput.equalsIgnoreCase("exit")) {
+                break;
+            }
+
+            // see tasks
+            if (userInput.equalsIgnoreCase("tasks")) {
+                System.out.println("lubot:");
+                for (int i=0; i<n; i++) {
+                    System.out.println(String.format("  %d: %s", i+1, tasks.get(i)));
+                }
+				System.out.println(horizontalBar);
+                continue;
+            }
+
+			// mark or unmark task
+            if (userInput.toLowerCase().startsWith("mark") || userInput.toLowerCase().startsWith("unmark")) {
+				markUnmarkTask(userInput);
+				System.out.println(horizontalBar);
+				continue;
+			}
+
+			// delete task
+            if (userInput.toLowerCase().startsWith("delete")) {
+				deleteTask(userInput);
+				System.out.println(horizontalBar);
+				continue;
+			}
+
+			// task
+			computeTask(userInput);
+			System.out.println(horizontalBar);
+        }
+
+        // end
+        System.out.println("lubot: pls come again");
+        System.out.println(horizontalBar);
+
+    }
+	
+	private static void markUnmarkTask(String userInput) {
+		// format input
+		int number;
+		String[] userInputs = userInput.split(" ", 2);
+		
+		// check 2nd argument
+		try {
+			number = Integer.parseInt(userInputs[1]) - 1;
+		} catch (NumberFormatException e) {
+			System.out.println("lubot: invalid input, pls type an int after mark");
+			return;
+		}
+
+		if (number < 0 || number > n-1) {
+			System.out.println("lubot: invalid task no, pls enter a number from 1 to " + n);
+			return;
+		}
+
+		// mark task
+		if (userInputs[0].equalsIgnoreCase("mark")) {
+            tasks.set(number, tasks.get(number).markDone());
+			System.out.println("lubot: ive marked the following task!");
+			System.out.println(String.format("  %d: %s", number+1, tasks.get(number)));
+			return;
+		}
+
+		// unmark task
+		if (userInputs[0].equalsIgnoreCase("unmark")) {
+            tasks.set(number, tasks.get(number).markUndone());
+			System.out.println("lubot: ive unmarked the following task!");
+			System.out.println(String.format("  %d: %s", number+1, tasks.get(number)));
```

Comment:
Should operators should be surrounded by a space character? I noticed the same issue in several other places too

```suggestion
			System.out.println(String.format("  %d: %s", number + 1, tasks.get(number)));
```